File: Python/Code/1900/1924 - 2007년.py
a, b = map(int, input().split())
month = [0, 31,28,31,30,31,30,31,31,30,31,30,31]
while a > 0:
	a -= 1
	b += month[a]
b -= 1
day = ["MON", "TUE", "WED", "THU", "FRI", "SAT", "SUN"]
print(day[b % 7])

File: Java/Code/1900/1924 - 2007년.java
import java.util.Scanner;

public class Main {
	public static void main(String[] args) {
		String[] result = { "SUN", "MON", "TUE", "WED", "THU", "FRI", "SAT" };
		int[] dayCount = { 0, 31, 28, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31 };

		Scanner sc = new Scanner(System.in);
		int month = sc.nextInt(), day = sc.nextInt();
		while (month > 0) {
			month--;
			day += dayCount[month];
		}
		System.out.println(result[day % 7]);
		return;
	}
}